packages/components/CHANGELOG.md

```diff
@@ -18,6 +18,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - `Stepper Horizontal` matches new design language.
 
+#### Changed
+
+- Use calc() instead of max() to calculate padding for `gridStyle` and `pds-grid`
```

Comment:
I think context is missing here. Perhaps we should prefix this with something like `Styles: ...` just like we usually do for other packages e.g. `React: ..`.